packer/Makefile

```diff
@@ -0,0 +1,18 @@
+SHELL += -eu
+
+BLUE	:= \033[0;34m
+GREEN	:= \033[0;32m
+RED   := \033[0;31m
+NC    := \033[0m
+
+# ∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨∨
+
+export AZURE_LOCATION       		 			?= West US 2
```

Comment:
not sure these should be hardcoded in the makefile (they should be set as env vars in CI)

Reply:
agreed.